src/Omnipay/Common/Helper.php

```diff
@@ -33,6 +34,26 @@ function ($match) {
     }
 
     /**
+     * Convert strings with underscores to be all lowercase before camelCase is preformed.
+     *
+     * @param  string $str The input string
+     * @return string The output string
+     */
+    protected static function convertToLowercase($str)
```

Comment:
What does this do that `strtolower($str)` does not do? Apart from not changing the case if the string does not contain any underscores.